### PR TITLE
feat: stats added

### LINKED ISSUE #37: Add stats page
The subgraph has some totals / statistics. These could be visualised on a stats page

### PR DESCRIPTION
**Pull request by sadda11asm**
closes #37
Here is simply how it looks like:
image

### CODE REVIEW

File: packages/react-app/src/Stats.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+                                    emoji={"🖼️"}
+                                />
+                                <StatCard
+                                    name={"Tokens"}
+                                    value={totalData.tokens}
+                                    emoji={"🪙"}
+                                />
+                                <StatCard
+                                    name={"Sale Value"}
+                                    value={totalData.saleValue.toFixed(2)}
```
Comment (azf20) [inline packages/react-app/src/Stats.js]: I got an error if I went to the Leaderboard page, then switched to stats:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'toFixed')
```

File: packages/react-app/src/NftyWallet.js [hunk 2]
```diff
@@ -252,6 +253,17 @@ export default function NftyWallet(props) {
...
           </Menu.SubMenu>
+          <Menu.Item key="stats">
+            <NavLink to={"/stats"}>
+              <span>
+                <span style={{ padding: 8 }} role="img" aria-label="stats">
+                  📊
+                </span>{" "}
+                stats
+              </span>{" "}
+              <Badge style={badgeStyle} count={displayBalance} showZero />
```
Comment (azf20) [inline packages/react-app/src/NftyWallet.js]: I don't think we need the `Badge` here (`displayBalance` is the tokens

File: packages/react-app/src/LineChart.js [hunk 3]
```diff
@@ -0,0 +1,201 @@
+import React, {Component} from "react";
+import "./LineChart.css";
+
+
+class LineChart extends Component {
```
Comment (azf20) [inline packages/react-app/src/LineChart.js:5]: Nice! This is cool, wondered why you didn't use a charting library?

File: packages/react-app/src/StatCard.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+const StatCard = props => {
```
Comment (azf20) [inline packages/react-app/src/StatCard.js:3]: Would be cool to also import this into the Artist.js page as I think it's now a common component

**Review by azf20 — CHANGES_REQUESTED** [review]
This is great! I think just one more guard required (I get a type error when navigating to the page sometimes). Also removing the badge from the navigation. But then I think this is good to go!

File: packages/react-app/src/LineChart.js (hunk 3 above)
Comment (sadda11asm) [reply packages/react-app/src/LineChart.js:5]: I already worked with this one so decided to choose it here. Is that okay? 😅

**Review by azf20 — APPROVED** [review]
Nice, thanks!

**Pull request merged**